src/docs/SUPPORTED_MODELS.md

```diff
@@ -46,6 +46,18 @@
     </tr>
     <tr>
       <td rowspan="3" vertical-align="top"><code>LlamaForCausalLM</code></td>
```

Comment:
Issue with table layout after adding new row for Llama 3

```suggestion
      <td rowspan="4" vertical-align="top"><code>LlamaForCausalLM</code></td>
```

Reply:
Applied